.github/workflows/ci.yml

```diff
@@ -5,41 +5,38 @@ on: [push, pull_request]
 jobs:
   format:
     name: Code formatting
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
The reason of this it that only `ubuntu-20.04` support OTP 22 https://repo.hex.pm/builds/otp/ubuntu-20.04/builds.txt